ci/publish

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
+
+if [ "$1" ]
```

Comment:
are you checking if `$1` exists, or if it is an empty string? 
I would avoid checking for unbound variables in this way, it is more safe to use `/bin/bash -e -u` (see the `-u`) and here check the number of input arguments `if [ $# -gt 0 ]` and you can also check if it is not an empty string with the most maniacal `if [ -n "${1}" ]`

Reply:
Done :-)